### DISCUSSION TITLE
What are all of the possible pre and postCommands?

### DISCUSSION THREAD

**Discussion by futuremotiondev** [ArturoDent/find-and-transform#32] [Q&A] [answered]
The README doesn't list all the possible `preCommands` and `postCommands`. Is there a list somewhere? I tried rifling through the source code but couldn't find anything.

Would greatly appreciate some help!

**Comment by ArturoDent** [comment, marked as answer]
From the README

> Use the commands from vscode's Keyboard Shortcuts context menu and `Copy Command ID` - the same command ID's you would use in a keybinding.  And the same for the `args` of each of those commands - see the `type` example above. 

You can use the commands you see in the Keyboard Shortcuts editor.  There may be some exceptions if they have complicated arguments.  The great majority of those commands take no arguments.
